app/views/admin/divisions/_divisions_grid_definition.html.slim

```diff
@@ -1,5 +1,5 @@
 = define_grid(@divisions_grid, show_filters: :show, hide_csv_button: true,
-  html: { class: 'rowlink table-hover', data: { link: @csv_mode ? nil : 'row' } }) do |g|
+  html: { class: 'rowlink table-hover', data: { link: 'row' } }) do |g|
```

Comment:
Do you prefer `{class: 'class'}` or `{ class: 'class' }` for syntax? I've kept each of these files with the same syntax they were before I edited them.